addons/udes_stock/tests/test_stock_quant.py

```diff
@@ -83,7 +83,7 @@ def test_02_gather_success(self):
         self.assertEqual(gathered_items.mapped('product_id'), self.apple)
 
         # Unfold the returned quants
-        first_quant, second_quant, third_quant = gathered_items
+        _, second_quant, _ = gathered_items
```

Comment:
`_` is used by Odoo. We usually add `_` to the parameter, like: `_first_quant`. If you want to shorten it, happy with `_q1` and `_q3`